doc/source/cli_accuracy.md

```diff
@@ -0,0 +1,384 @@
+(cli-accuracy)=
+
+# Accuracy workflow
+
+## Summary
+
+The accuracy workflow is designed to help users analyze differences between two elevation datasets by
+generating various outputs. It also includes optional coregistration for improved alignment.
+
+:::{figure} imgs/accuracy_workflow_pipeline.png
+:width: 100%
+:::
+
+## Available command line
+
+Run the workflow:
+
+```{code}
+xdem accuracy --config config_file.yaml
+```
+To display a template of all available configuration options for the YAML file, use the following command:
+
+```{code}
+xdem accuracy --display-template-config
+```
+
+## Detailed description of input parameters
+
+```{eval-rst}
+.. tabs::
+
+   .. tab:: inputs
+
+      **Required:** Yes
+
+      Elevation input information.
+
+      .. tabs::
+
+        .. tab:: reference_elev
+
+            .. csv-table:: Inputs parameters for reference_elev
+               :header: "Name", "Description", "Type", "Default value", "Required"
+               :widths: 20, 40, 20, 10, 10
+
+               "path_to_elev", "Path to reference elevation", "str", "", "Yes"
+               "force_source_nodata", "No data elevation", "int", "", "No"
+               "path_to_mask", "Path to mask associated to the elevation", "str", "", "No"
+               "from_vcrs", "Original vcrs", "str, int", "EGM96", "No"
+               "to_vcrs", "Destination vcrs", "str, int", "EGM96", "No"
+
+            .. note:: For setting the vcrs please refer to :doc:`vertical_ref`
+
+        .. tab:: to_be_aligned_elev
+
+            .. csv-table:: Inputs parameters for to_be_aligned_elev
+               :header: "Name", "Description", "Type", "Default value", "Required"
+               :widths: 20, 40, 20, 10, 10
+
+               "path_to_elev", "Path to to_be_aligned elevation", "str", "", "Yes"
+               "force_source_nodata", "No data elevation", "int", "", "No"
+               "path_to_mask", "Path to mask associated to the elevation", "str", "", "No"
+               "from_vcrs", "Original vcrs", "int, str", "EGM96", "No"
+               "to_vcrs", "Destination vcrs", "int, str", "EGM96", "No"
+
+            .. note:: For setting the vcrs please refer to :doc:`vertical_ref`
+
+      .. code-block:: yaml
+
+        inputs:
+            reference_elev:
+                path_to_elev: "path_to/ref_dem.tif"
+                force_source_nodata: -32768
+                from_vcrs: "EGM96"
+                to_vcrs: "Ellipsoid"
+            to_be_aligned_elev:
+                path_to_elev: "path_to/to_be_aligned_dem.tif"
+                path_to_mask: "path_to/mask.tif"
+
+   .. tab:: coregistration
+
+      **Required:** No
+
+      Coregistration step details. You can create a pipeline with up to three coregistration steps by
+      using the keys step_one, step_two, and step_three.
+      Available coregistration : NuthKaab, DhMinimize, VerticalShift, DirectionalBias, TerrainBias
```

Comment:
We might forget to update this list in the future. Should we point to the right doc page that lists available algorithms?